src/test/groovy/hudson/plugins/gradle/GradlePluginIntegrationTest.groovy

```diff
@@ -271,6 +272,11 @@ task hello << { println 'Hello' }"""))
         installation.name == 'myGradle'
         installation.home == '/tmp/foo'
 
+        def envVars = new EnvVars()
+        installation.buildEnvVars(envVars)
+        assert envVars.containsKey("PATH+GRADLE")
+        assert envVars.get("PATH+GRADLE") == installation.home + "/bin"
```

Comment:
This is a unit test. An integration test would actually demonstrate that `$PATH` is right.